src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -507,16 +507,16 @@ ${f.resolution}
     return s.join('').trim();
   }
 
-  private getTourReport(findings: FindingToReport[]): string {
+  private getTourReport(findings: FindingToReport[], cmd: CommandInstance): string {
     const tourFindings: FindingTour = {
       title: "Upgrade Project",
       steps: []
     };
 
+    const project: Project = this.getProject(this.projectRootPath + '');
```

Comment:
Retrieving the project is slow and we shouldn't do it multiple times. Why not reuse the project instance retrieved earlier in the command?